db-scripts/src/main/resources/migration.sql

```diff
@@ -812,3 +812,41 @@ WHERE NOT EXISTS (SELECT * FROM genetic_profile WHERE  STABLE_ID=CONCAT(`fusion_
     AND CANCER_STUDY_ID = `fusion_studies`.CANCER_STUDY_ID);
 DROP TEMPORARY TABLE fusion_studies;
 UPDATE `info` SET `DB_SCHEMA_VERSION`="2.12.3";
+
+##version: 2.12.4
+CREATE TABLE `resource_definition` (
+  `RESOURCE_ID` varchar(255) NOT NULL,
+  `DISPLAY_NAME` varchar(255) DEFAULT NULL,
+  `DESCRIPTION` varchar(2048) NOT NULL,
```

Comment:
@dippindots @adamabeshouse any specific reason why `DISPLAY_NAME` could be null and `DESCRIPTION` not to be?